src/modules/iotjs_module_gpio.c

```diff
@@ -30,7 +30,7 @@ static iotjs_gpio_t* iotjs_gpio_create(const iotjs_jval_t* jgpio) {
   IOTJS_VALIDATED_STRUCT_CONSTRUCTOR(iotjs_gpio_t, gpio);
   iotjs_jobjectwrap_initialize(&_this->jobjectwrap, jgpio,
                                &this_module_native_info);
-#if defined(__linux__)
+#if defined(__LINUX__)
```

Comment:
Thank you for explanation. `__LINUX__' is confusing for that purpose. Please use appropriate name, not `__LINUX__`.